sound/soc/sof/intel/hda-dai.c

```diff
@@ -95,9 +95,10 @@ static int hda_link_dma_params(struct hdac_ext_stream *stream,
 			       struct hda_pipe_params *params)
 {
 	struct hdac_stream *hstream = &stream->hstream;
+	unsigned char stream_tag = hstream->stream_tag;
 	struct hdac_bus *bus = hstream->bus;
-	unsigned int format_val;
 	struct hdac_ext_link *link;
+	unsigned int format_val;
```

Comment:
just moving format_val definition one line below it was?   What's this for? 
Please go through the diff file to make sure you keep the changes necessary and minimum.

Reply:
@wenqingfu   It is a style in kernel :  Inverted triangle

Reply:
I am not a clean freak :)